.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,23 @@
+name: PR Build
+
+on:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '6.0.x'
```

Comment:
We probably don't need this, since a PR build will not end up going to Nuget. This would save 10s of the total 27s, 37%.